utils/layer_from_netCDF.py

```diff
@@ -188,7 +188,7 @@ def make_node_layer(ds, progress_bar=None):
     # create inverse mapping
 
     if 'node_mapping' not in ds.ds.variables:
-        progress_bar.increase_progress(10, "no node mapping found in netCDF, skip object mapping. Model only has 2d?")
+        progress_bar.increase_progress(10, "no node mapping found in netCDF, skip object mapping. Model only has 1d?")
```

Comment:
node_mapping is not added when the model has only 2d (I have a small test model with only 2d which triggered this case)